docs/add-ons/prometheus.md

```diff
@@ -8,20 +8,41 @@ This project provides support for installing a open source Prometheus server in
 
 The following will deploy the Prometheus server into an EKS Cluster and provision a new Amazon Managed Service for Prometheus instance.
 
-```hcl
+```hcl-terraform
 # Creates the AMP workspace and all the relevent IAM Roles
-enable_amazon_prometheus         = true
-amazon_prometheus_workspace_alias = "EKS-Metrics-Workspace"
+enable_amp_for_prometheus         = true
 
-# Deploys Pometheus server with remote write to AWS AMP Workspace
+# Deploys Prometheus server with remote write to AWS AMP Workspace
 enable_prometheus             = true
 ```
 
+Enable Prometheus with custom `values.yaml`
+
+```hcl-terraform
+  #---------------------------------------
+  # Prometheus Server integration with Amazon Prometheus
+  #---------------------------------------
+  enable_amp_for_prometheus = true
+  # amazon_prometheus_workspace_id = module.aws-eks-accelerator-for-terraform.amazon_prometheus_workspace_id # Optional variable to use existing AMP workspace or it will create a new one
+  enable_prometheus = true
+  # Optional Map value
+  prometheus_helm_config = {
+    name       = "prometheus"                                         # (Required) Release name.
```

Comment:
Do we need to advanced config here?

Reply:
I thought our plan is to update the docs with basic usage and advanced usage of each add-on? Is there any place where we can document this?

Reply:
Fine by me